packages/office-ui-fabric-react/src/components/Announced/examples/Announced.QuickActions.Example.tsx

```diff
@@ -0,0 +1,289 @@
+  name: string;
+  modified: string;
+  modifiedby: string;
+  filesize: string;
+}
+
+export interface IAnnouncedQuickActionsExampleState {
+  items: IAnnouncedQuickActionsExampleItem[];
+  selectionDetails: {};
+  showItemIndexInView: boolean;
+  renameDialogOpen: boolean;
+  dialogContent: JSX.Element | undefined;
+  announced?: JSX.Element;
+}
+
+export class AnnouncedQuickActionsExample extends React.Component<{}, IAnnouncedQuickActionsExampleState> {
+  private _selection: Selection;
+  private _detailsList = createRef<IDetailsList>();
+  private _textField = createRef<ITextField>();
+
+  constructor(props: {}) {
+    super(props);
+
+    // Populate with items for demos.
+    if (_items.length === 0) {
+      for (let i = 0; i < 20; i++) {
+        _items.push({
+          key: i,
+          name: 'Item ' + i,
+          modified: generateRandomDate(),
+          modifiedby: _names[Math.floor(Math.random() * _names.length)],
+          filesize: Math.floor(Math.random() * 30).toString() + ' MB'
+        });
+      }
+    }
+
+    this._selection = new Selection({
+      onSelectionChanged: () => this.setState({ selectionDetails: this._getSelectionDetails() })
+    });
+
+    this._onRenderRow = this._onRenderRow.bind(this);
+    this._deleteItem = this._deleteItem.bind(this);
+    this._onRenderItemColumn = this._onRenderItemColumn.bind(this);
+    this._closeRenameDialog = this._closeRenameDialog.bind(this);
+    this._renderAnnounced = this._renderAnnounced.bind(this);
+
+    this.state = {
+      items: _items,
+      selectionDetails: this._getSelectionDetails(),
+      showItemIndexInView: false,
+      renameDialogOpen: false,
+      dialogContent: undefined,
+      announced: undefined
+    };
+  }
+
+  public render(): JSX.Element {
+    const { items, renameDialogOpen, dialogContent } = this.state;
+
+    return (
+      <>
+        {this._renderAnnounced()}
+        <MarqueeSelection selection={this._selection}>
+          <DetailsList
+            componentRef={this._detailsList}
+            items={items}
+            columns={_columns}
+            setKey="set"
+            layoutMode={DetailsListLayoutMode.fixedColumns}
+            selection={this._selection}
+            selectionPreservedOnEmptyClick={true}
+            ariaLabelForSelectionColumn="Toggle selection"
+            ariaLabelForSelectAllCheckbox="Toggle selection for all items"
+            onRenderItemColumn={this._onRenderItemColumn}
+            onRenderRow={this._onRenderRow}
+          />
+          <Dialog hidden={!renameDialogOpen} onDismiss={this._closeRenameDialog} closeButtonAriaLabel="Close">
+            {dialogContent}
+          </Dialog>
+        </MarqueeSelection>
+      </>
+    );
+  }
+
+  public componentWillUnmount() {
+    if (this.state.showItemIndexInView) {
+      const itemIndexInView = this._detailsList!.current!.getStartItemIndexInView();
+      alert('unmounting, getting first item index that was in view: ' + itemIndexInView);
```

Comment:
Do you still want this `window.alert`? I think they are disruptive for folks using assistive technology because they grab focus. I'd remove if unnecessary.